membership-attribute-service/app/models/DynamoSupporterRatePlanItem.scala

```diff
@@ -5,7 +5,7 @@ import org.joda.time.LocalDate
 import play.api.libs.json.{Writes, __}
 
 object DynamoSupporterRatePlanItem {
-    implicit val currencyWrite: Writes[Currency] = __.write[String].contramap(_.iso)
+  implicit val currencyWrite: Writes[Currency] = __.write[String].contramap(_.iso)
```

Comment:
it insisted on a bunch of formatting changes which have nothing to do with my actual change...